invenio_records_resources/services/base/service.py

```diff
@@ -48,7 +48,7 @@ def require_permission(self, identity, action_name, **kwargs):
         Like `check_permission` but raises an error if not allowed.
         """
         if not self.check_permission(identity, action_name, **kwargs):
-            raise PermissionDeniedError(action_name)
+            raise PermissionDeniedError(action_name, kwargs["record"])
```

Comment:
Needed it for this https://github.com/inveniosoftware/invenio-app-rdm/pull/2670/files#diff-2fb602eed821cbd7d07f7c61560b3e8ed50e165259da085c113fff47274b8496R426